kf_lib_data_ingest/etl/load/load.py

```diff
@@ -72,14 +69,10 @@ def __init__(
         """
         super().__init__(cache_dir)
         self.target_api_config = TargetAPIConfig(target_api_config_path)
-        self.concept_targets = {
-            v["standard_concept"].UNIQUE_KEY: k
-            for k, v in self.target_api_config.target_concepts.items()
-        }
-        self.target_id_key = (
-            self.target_api_config.contents.target_service_entity_id
+        self._validate_entities(
```

Comment:
Can we also add a comment in `templates/my_study/ingest_package_config.py` that says the values in `target_service_entities` should come from target API config classes' `class_name`?

Reply:
√